guides/source/active_record_querying.md

```diff
@@ -1403,8 +1403,9 @@ WHERE people.name = 'John'
 LIMIT 1
 ```
 
-NOTE: Remember that, if `find_by` returns more than one registry, it will take
-just the first and ignore the others. Note the `LIMIT 1` statement above.
+NOTE: Note that if a query matches multiple records, `find_by` will
+fetch only the first one and ignore the rest (see the `LIMIT 1`
```

Comment:
I just thought that it reads better but I might be wrong! Do you think that "the others" is more suitable?